.github/workflows/test.yml

```diff
@@ -17,13 +17,13 @@ jobs:
     # needs: lint
     strategy:
       matrix:
-        ckan-version: [2.8]
+        ckan-version: [2.9]
```

Comment:
version is hard-coded in a number of places, should we remove this setting?

Reply:
Removed matrix stuff.